beacon-chain/powchain/log_processing.go

```diff
@@ -33,6 +54,14 @@ func (w *Web3Service) ProcessLog(depositLog gethTypes.Log) {
 	// Process logs according to their event signature.
 	if depositLog.Topics[0] == hashutil.HashKeccak256(depositEventSignature) {
 		w.ProcessDepositLog(depositLog)
+		if !w.chainStarted {
+			triggered := w.isValidGenesisState(uint64(len(w.chainStartDeposits)), 0)
```

Comment:
you are passing `0` as `currentTime`, how will the chain ever start?

Reply:
missed it.
thanks